### DISCUSSION TITLE
Require less reviews for private modpacks

### DISCUSSION THREAD

**Discussion by ardera** [modrinth/.github#266] [Feature requests]
First of all, thanks to you all for this cool project. It's great work, and I really want to use it as my main launcher.

However, my personal modrinth experience has not been without issues. I'm playing on a private server with just 2 friends, and we're trying to add some mods to our current game. For that, we wanted to use Modrinth, since it seems to be the most modern, well-rounded modded minecraft launcher/platform that exists. But let me share some of the problems I had:

The idea was to just copy a public modpack on the modrinth platform and modify it a bit, adding / removing some mods.

1. Created the modpack project on the website, added the modpack file, added a summary (that I need to specify for a modpack with 3 users, for whatever reason)
2. Apply for publishing as "unlisted".
3. AutoMod rejects the modpack because of copyright violations. Even though those same mods are in the modpack I just downloaded on modrinth, and it seemed to be fine there.
4. remove ALL the listed violating mods. export, upload again. no new mods added.
5. Surprise, AutoMod rejects some different mods now!
6. remove all the listed violating mods again. export, upload again. no new mods added. Keep in mind, all the time I have in the back of my mind that modrinth might perma-ban me, because that's what I'm being threatened with in the "Project status" widget, for privately distributing the exact same modpack that is also publicly distributed on the modrinth website.
7. 3 days later, a real mod rejects my modpack, because the summary is not descriptive enough.
8. That makes sense, I just put an empty 5-word sentence, because this is a modpack intended to be published as unlisted and shared with exactly __2 people__ other than me, none of which will ever read the summary.

All this begs the questions: Why does modrinth apply the same evaluation criteria for private projects compared to public projects? In my case, I actually tried to "publish" it as unlisted, so I can set up my server without authenticating to modrinth. Does publishing it as private have lower bars of entry? If yes, is that mentioned anywhere? Are modpacks for small private servers just not the main target group of modrinth? Also why does respinning a public modpack in general cause so much friction?

Best regards,
Hannes

**Comment by blryface** [comment]
The issue is you can simply set it to listed then.
There's the option to withhold the project which will act more or less the same as an unlisted pack, and iirc the content moderators are supposed to do that, especially if you specifically state that this shouldn't be listed unless resent to submission

**Comment by blryface** [comment]
> that I need to specify for a modpack with 3 users, for whatever reason

You're not special, your project needs to follow all requirements regardless if you're only publishing it for 3 people

> AutoMod rejects the modpack because of copyright violations. Even though **those same mods are in the modpack I just downloaded on modrinth**, and it seemed to be fine there.

You need permission from the copyrighted mods authors to be able to specifically add it to your modpack.

> Keep in mind, all the time I have in the back of my mind that modrinth might perma-ban me, because that's what I'm being threatened with in the "Project status" widget, for privately distributing the exact same modpack that is also publicly distributed on the modrinth website.

Where did you get that info???

**Comment by AnOpenSauceDev** [comment]
What's stopping you from just creating a tiny .mrpack file with all the mods? If you don't intend to distribute the modpack to the public, then you probably don't need Modrinth.

I've published projects as unlisted with the intent to make them public after I finish working on assets like logos. Just because your modpack will become unlisted when approved doesn't mean you can't just instantly change it to public.

**Reply by ghost** [reply]
.mrpack require to create new profile, you will lose all your in game settings and have to wait downloading of all files again

**Reply by AnOpenSauceDev** [reply]
You can just copy over your `options.txt` and config folder from said profile, which only takes a few seconds.

If your launcher doesn't cache things, you may have to redownload mods, but in the grand scheme of things this isn't really that big of an issue, and I deal with this all the time. 

I know that some countries do have extremely slow throttled download speeds for sites like Modrinth, but that speed penalty is also there from updating a modpack, which would also require a large time downloading due to the changes in mods. If you don't like the waiting time from a .mrpack, you also should be able to get the pack creator to bake the mods via an override and send it directly via email/usb/whatever.

**Reply by ghost** [reply]
AnOpenSauceDev, if something is possible but inconvenient, you dont need to improve it, ok.

**Reply by blryface** [reply]
~~why am I "subscribed" to this~~
honestly, @LeFias , you're in the wrong here. You not knowing how something works doesnt mean it needs to be improved
